data/furniture/workspace_feeding_demo.urdf

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" ?>
```

Comment:
Please update this once the robot is moved (to wheelchair?) or to the side table.. For now, its fine.. :)